python-frontend/typeshed_serializer/tests/resources/fakemodule.pyi

```diff
@@ -1,4 +1,5 @@
 import sys
+import math as math
 from typing import overload
 from sys import flags as my_flags
 
```

Comment:
I think we miss a test with `from library import *`
Can we maybe add a `fakeImportedModule` and do `from fakeImportedModule import *` ?